src/coreclr/src/gc/gcpriv.h

```diff
@@ -4039,6 +4039,20 @@ class gc_heap
     PER_HEAP
     int generation_skip_ratio;//in %
 
+#ifdef FEATURE_CARD_MARKING_STEALING
+    PER_HEAP
+    VOLATILE(size_t) n_eph_soh;
+    PER_HEAP
+    VOLATILE(size_t) n_gen_soh;
+    PER_HEAP
+    VOLATILE(size_t) n_eph_loh;
+    PER_HEAP
+    VOLATILE(size_t) n_gen_loh;
+#endif //FEATURE_CARD_MARKING_STEALING
+
+    PER_HEAP_ISOLATED
+    int generation_skip_ratio_threshold; 
```

Comment:
format issue.

Reply:
what's the issue?

Reply:
Very trivial!  There is a blank after the `int generation_skip_ratio_threshold;`